src/__tests__/HospitalRun.test.tsx

```diff
@@ -72,98 +62,116 @@ describe('HospitalRun', () => {
         )
       })
 
-      it('should render the Dashboard when the user does not have read appointment privileges', async () => {
-        const { wrapper } = await setup('/appointments')
-        expect(wrapper.find(Dashboard)).toHaveLength(1)
+      it('should render the Dashboard when the user does not have read appointment privileges', () => {
+        render('/appointments')
+        const main = screen.getByRole('main')
+        expect(within(main).getByRole('heading', { name: /example/i })).toBeInTheDocument()
       })
     })
 
     describe('/labs', () => {
-      it('should render the Labs component when /labs is accessed', async () => {
+      it('should render the Labs component when /labs is accessed', () => {
         jest.spyOn(LabRepository, 'findAll').mockResolvedValue([])
         const permissions: Permissions[] = [Permissions.ViewLabs]
-        const { wrapper } = await setup('/labs', permissions)
-
-        expect(wrapper.find(ViewLabs)).toHaveLength(1)
+        render('/labs', permissions)
+
+        const table = screen.getByRole('table')
+        expect(within(table).getByText(/labs.lab.code/i)).toBeInTheDocument()
+        expect(within(table).getByText(/labs.lab.type/i)).toBeInTheDocument()
+        expect(within(table).getByText(/labs.lab.requestedOn/i)).toBeInTheDocument()
+        expect(within(table).getByText(/labs.lab.status/i)).toBeInTheDocument()
+        expect(within(table).getByText(/actions.label/i)).toBeInTheDocument()
```

Comment:
Thoughts about simplifying this check to only check that the correct header gets rendered? For example, check that there is an h1 on the page that has the correct value. 

Alternatively, could we do an assertion that the route remain unchanged or that the route is not equal to `/labs`

Reply:
I think that the same comment can be applied for the rest of the routes as well.

Reply:
> simplifying this check to only check that the correct header gets rendered?

> I think that the same comment can be applied for the rest of the routes as well.

To clarify the reasoing behind this, we only need to check the heading/route as there are tests covering the actual functionality of these pages elsewhere?